src/Components/Nav/Nav.js

```diff
@@ -33,10 +36,10 @@ export default function Nav() {
         <div className="nav">
             <ul className="nav__links">
                 <li>
-                    <a href="#All" onClick={filter}>ALL</a>
+                    <a href="#All" onClick={filter}>{t(I18N_KEYS.ALL)}</a>
                 </li>
                 {
-                    genres.slice(0, 6).map((genre) => {
+                    genres.map((genre) => {
```

Comment:
As already mentioned, it's better to extract such logic into to separate variable/functions: one that takes a list and maps through it, and the other one - takes a single item from the list and returns some stuff (JSX in this case).